storage/reader.go

```diff
@@ -142,12 +142,17 @@ func uncompressedByServer(res *http.Response) bool {
 func parseCRC32c(res *http.Response) (uint32, bool) {
 	const prefix = "crc32c="
 	for _, spec := range res.Header["X-Goog-Hash"] {
-		if strings.HasPrefix(spec, prefix) {
-			c, err := decodeUint32(spec[len(prefix):])
-			if err == nil {
-				return c, true
+		values := strings.Split(spec, ",")
```

Comment:
Can you speak about why this change is necessary? How does the header format differ between APIs?

Reply:
I added a comment to the function. 
The XML header looks something like: [crc32c=xxx md5=xxx] 
while the JSON header looks like:         [crc32c=xxx,md5=xxx]

So I added on a split on comma